plugins/account_history/plugin.cpp

```diff
@@ -90,6 +106,7 @@ if( options.count(name) ) { \
                     note.op.visit(operation_visitor(database, note, item));
                 }
             }
+            erase_old_blocks();
```

Comment:
Propose move this to block handler instead of operation handler. Block can contain several transactions and each transaction can contain 500+ operations. It's not efficient to call this on each operation.
@afalaleev, is it safe to move this to `applied_block` handler?

Reply:
I agree.

@RostislavVel  see lines https://github.com/GolosChain/golos/blob/668-Improve-history-ops-filters-usability/plugins/operation_history/plugin.cpp#L212-L214
And in the same way attach to `database::applied_block`-signal . And call `erase_old_block` from it.